federation/cmd/federated-controller-manager/app/options/options.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package options provides the flags used for the controller manager.
+
+package options
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apis/componentconfig"
+	"k8s.io/kubernetes/pkg/client/leaderelection"
+)
+
+type ControllerManagerConfiguration struct {
+	unversioned.TypeMeta
+
+	// port is the port that the controller-manager's http service runs on.
+	Port int `json:"port"`
+	// address is the IP address to serve on (set to 0.0.0.0 for all interfaces).
+	Address string `json:"address"`
+	// concurrentSubRCSyncs is the number of sub replication controllers that are
+	// allowed to sync concurrently. Larger number = more responsive replica
+	// management, but more CPU (and network) load.
+	ConcurrentSubRCSyncs int `json:"concurrentSubRCSyncs"`
+	// clusterMonitorPeriod is the period for syncing ClusterStatus in cluster controller.
+	ClusterMonitorPeriod unversioned.Duration `json:"clusterMonitorPeriod"`
+	// federatedAPIQPS is the QPS to use while talking with ubernetes apiserver.
+	FederatedAPIQPS float32 `json:"federatedAPIQPS"`
```

Comment:
APIServerQPS and APIServerBurst seem like better names to me.
I think it is fine to assume that APIServer in federation-controller-manager means federation-apiserver.